.github/workflows/platformDeploy.yml

```diff
@@ -420,7 +424,7 @@ jobs:
   postGithubComment:
     name: Post a GitHub comment when platforms are done building and deploying
     runs-on: ubuntu-latest
-    if: ${{ always() }}
+    if: ${{ always() && !cancelled() }}
```

Comment:
Discussed over slack but I think we should write a little more logic to figure out the last build (if canceled) to let the CP requestor know which version their request was completed in.

Reply:
Good catch. Put something together to address that